package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "wpcom",
   "description": "Official JavaScript library for the WordPress.com REST API",
-  "version": "4.7.0",
+  "version": "4.7.1",
+  "main": "index.min.js",
```

Comment:
I don't think we should point to a minified file for the entry point. If desired to be minified, this should be the responsibility of the consuming code.